src/default_panels/StyleTracesPanel.js

```diff
@@ -196,32 +213,19 @@ const StyleTracesPanel = ({localize: _}) => (
         ]}
       />
     </TraceTypeSection>
-
     <TraceTypeSection
       name={_('Text')}
-      traceTypes={['scatter', 'scatterpolar', 'scatterpolargl']}
+      traceTypes={['scatter', 'scatterpolar', 'scatterpolargl', 'pie']}
     >
       <FontSelector label={_('Typeface')} attr="textfont.family" />
       <Numeric label={_('Font Size')} attr="textfont.size" units="px" />
       <ColorPicker label={_('Font Color')} attr="textfont.color" />
-      <Dropdown
+      <TextPosition
         label={_('Text Position')}
         attr="textposition"
-        clearable={false}
-        options={[
-          {label: _('Top Left'), value: 'top left'},
-          {label: _('Top Center'), value: 'top center'},
-          {label: _('Top Right'), value: 'top right'},
-          {label: _('Middle Left'), value: 'middle left'},
-          {label: _('Middle Center'), value: 'middle center'},
-          {label: _('Middle Right'), value: 'middle right'},
-          {label: _('Bottom Left'), value: 'bottom left'},
-          {label: _('Bottom Center'), value: 'bottom center'},
-          {label: _('Bottom Right'), value: 'bottom right'},
-        ]}
+        localize={_}
```

Comment:
Why are we passing in localize? If someone is using our API to build panels and they use `TextPosition` how will they know to pass in `localize`? No other component has this pattern as far as I am aware.